dex/router/src/config.rs

```diff
@@ -0,0 +1,69 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{enable_swap_by_user::EnableSwapByUserConfig, factory::PairTokens};
+
+#[multiversx_sc::module]
+pub trait ConfigModule {
+    fn is_active(&self) -> bool {
+        self.state().get()
+    }
+
+    fn check_is_pair_sc(&self, pair_address: &ManagedAddress) {
+        require!(
+            self.address_pair_map().contains_key(pair_address),
+            "Not a pair SC"
+        );
+    }
+
+    #[view(getPairCreationEnabled)]
+    #[storage_mapper("pair_creation_enabled")]
+    fn pair_creation_enabled(&self) -> SingleValueMapper<bool>;
+
+    #[view(getState)]
+    #[storage_mapper("state")]
+    fn state(&self) -> SingleValueMapper<bool>;
```

Comment:
A better naming would make using this more easily.
For example, if `state = false` what does this mean?

Reply:
Indeed, I agree that the naming isn't the most fortunate, but this is how it was before. I only moved the code in a different module, to be able to call some functions in the multi pair swap module.